misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -0,0 +1,52 @@
+package misk.redis
+
+import misk.time.FakeClock
+import java.time.Instant
+import javax.inject.Inject
+
+class FakeRedis : Redis {
+  @Inject lateinit var clock: FakeClock
+
+  // The value type stored in our key-value store
+  private data class Value (
+    val value: String,
+    val expiryInstant: Instant = Instant.MAX
+  )
+
+  // Acts as the Redis key-value store
+  private val keyValueStore = HashMap<String, Value>()
+
+  override fun del(key: String): Long {
```

Comment:
why does `del` return 0 or 1? seems like a boolean is more appropriate

Reply:
Its how Jedis' `del` command works (http://xetorthio.github.io/jedis/redis/clients/jedis/Jedis.html#del-java.lang.String-), it returns an integer telling you how many keys were deleted. I'll add a comment to make this clearer.

Reply:
can it ever be more than 1 if you pass a single key?

Reply:
k. to me a `fun del(key: String): boolean` makes more sense then

Reply:
No arguments against that, changed `del` to return `Boolean`